### PR TITLE
#5085 mobile fixes

### LINKED ISSUE #5085: Explore mobile v1.0.1
Here is a follow up for minor issues outstanding, mainly visual changes

## Icons
image
- should look like this, instead of sort there should be icon as well 
- burger menu should be moved to the bottom row and have stroke and shadow 
- if there is active filter we will add the same tag (count) to the icon as currently with sort by
image
- dont know how hard it is but maybe also stretch the other buttons so there is not gap and the screen is filled (in this case network button scaling till sort by is on the right side

## Animation 
- add slide in animation - From left would be the best since the button is there

## Menu 
- few visual changes here
image
- change the title to 16px bold - when there is no distinction it could confused for filter section
image
- remove dropshadow from reset all, it has too much contrast imo

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Closes #5085
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2023-03-01 at 8 39 58 PM

### CODE REVIEW

**Comment by prachi00** [comment]
Leftovers:

- [ ] Animation

**Comment by daiagi** [comment]
I'm doing quite similar work on #5076

File: components/explore/ExploreIndex.vue [hunk 1]
```diff
@@ -2,6 +2,7 @@
   <div class="explore is-flex is-flex-wrap-wrap">
     <ExploreTabItem />
     <div class="explore-menu is-flex">
+      <ExploreSidebarToggle v-if="isMobileDevice" />
```
Comment (daiagi) [inline components/explore/ExploreIndex.vue:5]:
```suggestion
<ExploreSidebarToggle class="is-hidden-tablet" />
```

File: components/explore/ExploreIndex.vue [hunk 2]
```diff
@@ -16,6 +17,8 @@ import ExploreSort from './ExploreSort.vue'
 import ExploreChain from './ExploreChain.vue'
 import ExploreGrid from './ExploreGrid.vue'
 import ExploreOffer from './ExploreOffer.vue'
+import { isMobileDevice } from '@/utils/extension'
```
Comment (daiagi) [inline components/explore/ExploreIndex.vue:20]: remove and use bulma class instead

File: components/explore/ExploreSort.vue [hunk 3]
```diff
@@ -47,6 +46,7 @@
 <script setup lang="ts">
 import { ODropdown } from '@oruga-ui/oruga'
 import { NeoButton, NeoDropdownItem } from '@kodadot1/brick'
+import { isMobileDevice } from '@/utils/extension'
```
Comment (daiagi) [inline components/explore/ExploreSort.vue:49]: isMobileDevice responds to user-agent and not to screen size
suggested to use bulma classes instead

https://bulma.io/documentation/helpers/visibility-helpers/#hide

File: components/explore/ExploreTabItem.vue [hunk 4]
```diff
@@ -1,22 +1,6 @@
...
-      </a>
-      <a
-        class="is-hidden-tablet"
-        :class="{ disabled: disabledTab }"
-        @click="openMobileFilters">
-        <b-icon :icon="'bars'" size="is-medium" />
-      </a>
-    </div>
-
+    <ExploreSidebarToggle v-if="!isMobileDevice" class="mr-4" />
```
Comment (daiagi) [inline components/explore/ExploreTabItem.vue:3]:
```suggestion
<ExploreSidebarToggle class="mr-4 is-hidden-mobile" />
```

File: styles/global.scss [hunk 5]
```diff
@@ -270,3 +270,7 @@ hr {
     }
   }
 }
+
```
Comment (daiagi) [inline styles/global.scss:273]: use bulma class ```is-shadowless``` instead

**Review by daiagi — CHANGES_REQUESTED** [review]

**Comment by exezbcz** [comment]
hmm, cannot make to look like the screenshot you added
my version looks like this
image

**Comment by prachi00** [comment]
> hmm, cannot make to look like the screenshot you added my version looks like this image

can you try clearing cache? it shows up fine for me
Screen Shot 2023-03-02 at 7 33 25 PM

**Comment by daiagi** [comment]
> Leftovers:
> 
> - [ ] Animation
> 

 Covered in #5158

File: components/explore/ExploreSort.vue [hunk 6]
```diff
@@ -64,6 +64,8 @@ const options = computed(() => {
     ? NFT_SQUID_SORT_CONDITION_LIST
     : NFT_SQUID_SORT_COLLECTIONS
 })
+const label = computed(() => (isMobileDevice ? '' : 'Sort By'))
+const icon = computed(() => (isMobileDevice ? 'filter' : ''))
```
Comment (daiagi) [inline components/explore/ExploreSort.vue:68]: As I said, I don't like the use of isMobileDevice beacsue it doesn't actually check screen size
So you could have unexpected results for bigger screen that are touch

I  would write the button twice once with is-hidden-mobile and once with is-hidden-tablet classes to toggle the views

If you don't like that approach then I would do something like this 

```
Const width =ref(window.innerWidth) 
onMounted(() => { 
   window.addEventListener('resize', () => { 
     width.value = window.innerWidth 
   }) 
 }) 
  
  
 const isMobile = computed(() => width.value <= 768) 
 ```

**Review by daiagi — CHANGES_REQUESTED** [review]

File: components/explore/ExploreSort.vue (hunk 6 above)
Comment (roiLeo) [reply components/explore/ExploreSort.vue:68]: > I would write the button twice, once with is-hidden-mobile and once with is-hidden-tablet classes to toggle the views

^ this

**Comment by daiagi** [comment]
It seems #5077 didn't quite resolve the issue of unequel cards height

**Review by daiagi — APPROVED** [review]

**Comment by exezbcz** [comment]
image
nice job! 
can you please switch the order of the buttons, so filter button is right, network middle and menu left? 

Thanks!

**Comment by daiagi** [comment]
> image
> nice job! 
> can you please switch the order of the buttons, so filter button is right, network middle and menu left? 
> 
> Thanks!  

There's going to be quite a merge with my pr anyways, I can take care of the button order then

**Comment by exezbcz** [comment]
@daiagi 
> if there is active filter we will add the same tag (count) to the icon as currently with sort by

this is also missing, can you add this as well?

**Comment by daiagi** [comment]
> this is also missing, can you add this as well? 


Sure, for mobile only or mobile and desktop?

**Comment by exezbcz** [comment]
@daiagi on desktop there isn't a burger menu styled with drop shadow etc + there are breadcrumbs indicating active filters, so only for devices, breakpoints, where breadcrumbs are turned off

**Comment by daiagi** [comment]
> @daiagi on desktop there isn't a burger menu styled with drop shadow etc + there are breadcrumbs indicating active filters, so only for devices, breakpoints, where breadcrumbs are turned off 

Yep, thought so. Just wanted to be clear

**Review by roiLeo — APPROVED** [review]
Let it roll

**Comment by vikiival** [comment]
@prachi00 please resolve conflicts

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
CC @prachi00 
Screenshot 2023-03-07 at 15 31 55

**Comment by prachi00** [comment]
I am just gonna close this one, it has too many conflicts, I'll see if any changes are remaining and do it in a new branch

**Pull request closed without merging**